src/lua/init.c

```diff
@@ -851,6 +843,15 @@ luaopen_tarantool(lua_State *L)
 #endif
 	lua_settable(L, -3);
 
+	/* build.asan */
+	lua_pushstring(L, "asan");
+#ifdef ENABLE_ASAN
+	lua_pushboolean(L, true);
+#else
+	lua_pushboolean(L, false);
+#endif
+	lua_settable(L, -3);
+
```

Comment:
Since this change is documented, it should have a changelog entry.

Reply:
Done.